docs/extending/kiotaabstractions.md

```diff
@@ -6,82 +6,107 @@ parent: Understand and Extend the Kiota generator
 
-```csharp
-    public interface IRequestAdapter {
-        void EnableBackingStore(IBackingStoreFactory backingStoreFactory);
+### Request Adapter
+
+The Request Adapter interface is the primary point where Kiota service libraries will trigger the creation of a HTTP request.  Below is the [C# implementation](https://github.com/microsoft/kiota/blob/main/abstractions/dotnet/src/IRequestAdapter.cs).
 
-        ISerializationWriterFactory SerializationWriterFactory { get; }
+```csharp
+public interface IRequestAdapter {
+    void EnableBackingStore(IBackingStoreFactory backingStoreFactory);
 
-        Task<ModelType> SendAsync<ModelType>(RequestInformation requestInfo,
-                                             IResponseHandler responseHandler = default)
-                     where ModelType : IParsable;
+    ISerializationWriterFactory SerializationWriterFactory { get; }
 
-        Task<IEnumerable<ModelType>> SendCollectionAsync<ModelType>(RequestInformation requestInfo,
-                        IResponseHandler responseHandler = default)
+    Task<ModelType> SendAsync<ModelType>(RequestInformation requestInfo,
+                                            IResponseHandler responseHandler = default)
                     where ModelType : IParsable;
 
-        Task<ModelType> SendPrimitiveAsync<ModelType>(RequestInformation requestInfo,
-                                                      IResponseHandler responseHandler = default);
+    Task<IEnumerable<ModelType>> SendCollectionAsync<ModelType>(RequestInformation requestInfo,
+                    IResponseHandler responseHandler = default)
+                where ModelType : IParsable;
 
-        Task SendNoContentAsync(RequestInformation requestInfo,
-                                IResponseHandler responseHandler = default);
-    }
+    Task<ModelType> SendPrimitiveAsync<ModelType>(RequestInformation requestInfo,
+                                                    IResponseHandler responseHandler = default);
+
+    Task SendNoContentAsync(RequestInformation requestInfo,
+                            IResponseHandler responseHandler = default);
+}
 ```
 
-Kiota service libraries return the model type that is associated with HTTP resource. This behavior can be overriden by changing the `responseHandler` to do something different than default behavior.  One use of this is to change the response type to be either a native HTTP response class, or return a generic API response class that provides access to more underlying metadata.
+Kiota service libraries return the model type that is associated with HTTP resource. This behavior can be overridden by changing the `responseHandler` to do something different than default behavior.  One use of this is to change the response type to be either a native HTTP response class, or return a generic API response class that provides access to more underlying metadata.
 
-## RequestInformation
+### Request Information
```

Comment:
Always use sentence case for section titles. (`Request information`)

Reply:
fixed!